cmake/defaults/Packages.cmake

```diff
@@ -209,9 +209,20 @@ if (PXR_BUILD_IMAGING)
     if (PXR_BUILD_OPENIMAGEIO_PLUGIN)
         set(REQUIRES_Imath TRUE)
         find_package(OpenImageIO REQUIRED)
+        set(OIIO_VERSION "${OpenImageIO_VERSION_MAJOR}.${OpenImageIO_VERSION_MINOR}.${OpenImageIO_VERSION_PATCH}")
         add_definitions(-DPXR_OIIO_PLUGIN_ENABLED)
+        if (TARGET OpenImageIO::idiff)
+            if (CMAKE_BUILD_TYPE)
+                string(TOUPPER "${CMAKE_BUILD_TYPE}" UPPER_BUILD_TYPE)
+                get_target_property(OIIO_idiff_BINARY OpenImageIO::idiff IMPORTED_LOCATION_${UPPER_BUILD_TYPE})
+            else()
+                get_target_property(OIIO_idiff_BINARY OpenImageIO::idiff IMPORTED_LOCATION)
+            endif()
+        endif()
         if (OIIO_idiff_BINARY)
             set(IMAGE_DIFF_TOOL ${OIIO_idiff_BINARY} CACHE STRING "Uses idiff for image diffing")
+        elseif (PXR_BUILD_TESTS)
+            message(FATAL_ERROR "Unable to find idiff for image diffing tests")
```

Comment:
Sorry for a slow response on this issue. If one builds imaging, builds the OIIO plugin, does wish to build tests, and hasn't got idiff (perhaps one did not build the binary tools, just the library), then cmake will fatal ~ wouldn't it make more sense to warn, not fatal that idiff based tests won't succeed? I think it's not sufficient that the idiff binary is found here, it also has to be true that the idiff binary is in the executable environment when the tests are run, and the cmake check isn't sufficient for that.

Reply:
Thanks for the comment and I'll make necessary changes.